Gemfile.lock

```diff
@@ -296,8 +292,8 @@ GEM
       sprockets-rails
     rails-deprecated_sanitizer (1.0.3)
       activesupport (>= 4.2.0.alpha)
-    rails-dom-testing (1.0.8)
-      activesupport (>= 4.2.0.beta, < 5.0)
+    rails-dom-testing (1.0.9)
```

Comment:
Is this version change expected?

Reply:
It was updated automatically when updating sass iirc. I'll re-check